patterns/1-initial/capacity-for-contributions.md

```diff
@@ -0,0 +1,74 @@
+## Title
+
+Managing capacity for reviewing contributions
+
+## Patlet
+
+Reviewing InnerSource contributions takes time and effort. This should be reflected in capacity planning, especially for larger contributions. Expectations and available capacity should be transparent so that contributors understand when their contributions will be reviewed and, if accepted, released.
+
+## Problem
+
+Large InnerSource contributions are causing delays to other work in the team and/or contributions are taking longer to be released than expected. Reviewing contributions may be significant invisible work, not tracked in a team's Agile development process.
+
+## Story
+
+The BBC's connected TV application are built by a number of teams, each with different areas of responsibility. They work on each other's areas of the codebase via InnerSource on a regular basis.
```

Comment:
A minor comment. Typically in patterns, the story is generalized to not mention a specific company or institution (it's a *pattern*, so it ideally should be described in ways that can be applied to other companies/users). So, maybe:
An application developed by a company for use by users outside the company is built by a number of teams, each with different areas of responsibility.

The specific information becomes apparent in the known instances reference.

Reply:
To get this pattern merged into the mainline I will let this one slide.
We rarely have stories in our patterns anyways, so i would rather have a too specific story than none :)

We can still fix this once we add the second known instance of this.